bootstrap.sh

```diff
@@ -23,24 +24,33 @@ last_bootstrapping_line_number=$(grep --line-number '^## Writing the Legislation
 
 cd ..
 pwd
-mv country-template openfisca-$lowercase_country_name
+mv country-template-master openfisca-$lowercase_country_name
 cd openfisca-$lowercase_country_name
 
 git init
 git add .
 git commit --message 'Initial import from OpenFisca country-template' --author='OpenFisca Bot <bot@openfisca.org>'
 
-sed -i '' "3,${last_bootstrapping_line_number}d" README.md  # remove instructions lines
-sed -i '' "s|country_template|$lowercase_country_name|g" README.md setup.py check-version-bump.sh Makefile `find openfisca_country_template -type f`
-sed -i '' "s|country-template|$lowercase_country_name|g" README.md
-sed -i '' "s|Country-Template|$COUNTRY_NAME|g" README.md setup.py check-version-bump.sh .github/PULL_REQUEST_TEMPLATE.md CONTRIBUTING.md `find openfisca_country_template -type f`
-sed -i '' "s|https://github.com/openfisca/openfisca-country-template|$URL|g" setup.py
+all_module_files=`find openfisca_country_template -type f`
+
+set -x
+
+# Use intermediate backup files due to lack of portable 'no backup' option. See https://stackoverflow.com/q/5694228/594053
+sed -i '.template' "s|country_template|$lowercase_country_name|g" README.md setup.py .circleci/config.yml Makefile $all_module_files
```

Comment:
This doesn't seem to work in `GNU/Sed`, I'm getting this error:

```
++ sed -i .template 's|country_template||g' README.md setup.py .circleci/config.yml Makefile openfisca_country_template/__init__.py openfisca_country_template/entities.py openfisca_country_template/parameters/benefits/basic_income.yaml openfisca_country_template/parameters/benefits/housing_allowance.yaml openfisca_country_template/parameters/general/age_of_majority.yaml openfisca_country_template/parameters/general/age_of_retirement.yaml openfisca_country_template/parameters/taxes/income_tax_rate.yaml openfisca_country_template/parameters/taxes/social_security_contribution.yaml openfisca_country_template/reforms/__init__.py openfisca_country_template/reforms/flat_social_security_contribution.py openfisca_country_template/reforms/modify_social_security_taxation.py openfisca_country_template/reforms/removal_basic_income.py openfisca_country_template/situation_examples/__init__.py openfisca_country_template/situation_examples/couple.json openfisca_country_template/situation_examples/single.json openfisca_country_template/tests/age.yaml openfisca_country_template/tests/basic_income.yaml openfisca_country_template/tests/disposable_income.yaml openfisca_country_template/tests/housing_allowance.yaml openfisca_country_template/tests/housing_tax.yaml openfisca_country_template/tests/income_tax.yaml openfisca_country_template/tests/reforms/modify_social_security_taxation.yaml openfisca_country_template/tests/situations/income_tax.yaml openfisca_country_template/tests/social_security_contribution.yaml openfisca_country_template/variables/benefits.py openfisca_country_template/variables/demographics.py openfisca_country_template/variables/housing.py openfisca_country_template/variables/income.py openfisca_country_template/variables/stats.py openfisca_country_template/variables/taxes.py
sed: -e expression #1, char 1: unknown command: `.'
```

Reply:
Yes, this is precisely what @michelbl was mentioning in https://github.com/openfisca/country-template/pull/47#issuecomment-393050065 (repeated below).